historical_main.py

```diff
@@ -0,0 +1,5 @@
+import asyncio
+from rs3_trading.historicaldata.historical_process import main
```

Comment:
Rename the main function since it's not really "main". Maybe history_process or get_history_data